### PR TITLE
Updates to Lab 8

### PR DESCRIPTION
**Pull request by borjasotomayor**
Last year's Lab 8 relied on a Redis example, and I think it's worth keeping the example (even though the project no longer uses Redis) because it's still a good example of code that requires a very specific software environment (plus an excuse to talk about Git submodules)

I am going to take care of running through all the steps in Part I, since I have to do so for the deployment lecture anyways, but someone should run through all the steps in Part II to make sure that there haven't been any changes in Heroku's interface, etc. Also, I've rewritten Part I to not assume that students are familiar with Redis, but you should pay special attention to anything that may seem unclear to anyone who has not used Redis before.

### CODE REVIEW

File: website/content/labs/lab8.md [hunk 1]
```diff
@@ -40,7 +42,9 @@ Once the fork has completed, you can clone it and begin your work. Remember to d
 Edit your tasks.txt file and include the URL of your forked cs220-redis-example repository on GitHub.
 
 {{% warning %}}
-Careful! Remember that your tasks.txt is in your personal repository on GitLab (with all your other lab files). You will only be using that repository to edit your tasks.txt file. The rest of your work will happen on the forked cs220-redis-example repository you just created.
+{{% md %}}
+Careful! Remember that your `tasks.txt` is in your personal repository on GitLab (with all your other lab files). You will only be using that repository to edit your `tasks.txt` file. The rest of your work will happen on the forked cs220-redis-example repository you just created.
+{{% /md %}}
 {{% /warning %}}
 
 ## Task 1: Running Docker in the CS VM [Optional]
```
Comment (mpcsh) [inline website/content/labs/lab8.md]: Can we link them to the headless guide in this section?

File: website/content/labs/lab8.md [hunk 2]
```diff
@@ -132,12 +132,16 @@ On a separate terminal, you can also try running a few Docker commands:
 ## Task 2: Git Submodules
 [20 points]
 
-Your `cs220-redis-example` repository has two directories:
+Your `cs220-redis-example` repository contains code that extends the Redis server with some additional commands. We will only be concerned with one of these commands: the `EXAMPLE.HGETSET` command. This commands operates on a hash table and takes two parameters: a key and a value. It will fetch the value currently associated with the given key, and will replace its value with the one provided as a parameter.
 
-* `module/` - This directory contains a Redis module that adds a few extra commands. We will be using just one of them: EXAMPLE.HGETSET, which does an HGET operation followed by an HSET operation. In other words, it gets the value of a key in a hash table (which will be returned by the command), and then sets the value of that key to a new value. Note: this example module is the same example module provided by Redis.
-* `tests/` - This directory contains a simple program that uses the hiredis library to connect to a Redis server to test the HGETSET operation.
+TODO: Include and example
```
Comment (mpcsh) [inline website/content/labs/lab8.md]: Was this intentional?

Comment (borjasotomayor) [reply website/content/labs/lab8.md]: Yes, the module/ and tests/ description were moved to a different point in the writeup. My latest commit adds an example here.

File: website/content/labs/lab8.md (hunk 1 above)
Comment (borjasotomayor) [reply website/content/labs/lab8.md]: I've added a link.

**Comment by borjasotomayor** [comment]
I've worked through all the examples in Part I and they all work (I've also removed a few unnecessary steps)

**Pull request merged**